tests/end_to_end/test_fim/test_fim_windows/data/test_cases/cases_fim.yaml

```diff
@@ -0,0 +1,41 @@
+- name: create_file_windows
```

Comment:
Rename the file, please.

Reply:
Done in 32249dfa1d732e79983d294b7848c8f12e53a5cf